examples/nixnet/lin-signal-single-point-output.py

```diff
@@ -4,6 +4,8 @@
  This is used to demonstrate a signal single point output session. 
  This example uses hardcoded signal names that use the NIXNET_example database. 
```

Comment:
This example uses `NIXNET_exampleLDF` database, not `NIXNET_example`. Looks like these comments were copied from one python example to another. There might be slight changes based on device, databases and mode you are using in each example. Compare comments from all example with equivalent C examples.

Reply:
Done.